src/lib.rs

```diff
@@ -97,7 +98,12 @@ pub fn formatter(
         FormatterError::Reading(ReadingError::Io("Failed to read query content".into(), e))
     })?;
 
-    let configuration = Configuration::parse(&query)?;
+    let mut configuration = Configuration::parse(&query)?;
+    // Replace the language deduced from the query file by the one from the CLI, if any
+    match language {
```

Comment:
I think `if let Some(...) = ...` reads nicer here.

Reply:
Indeed, thanks. I did that in the new version.